src/main/java/com/revature/personalfinance/service/Authenticator.java

```diff
@@ -3,32 +3,65 @@
 import org.springframework.context.annotation.Scope;
 import org.springframework.stereotype.Component;
 
+import com.google.firebase.auth.FirebaseAuth;
+import com.google.firebase.auth.FirebaseAuthException;
+import com.google.firebase.auth.FirebaseToken;
+
 import lombok.Data;
 
 @Component
 @Scope("singleton")
 @Data
 public class Authenticator {
-	
+
 	/**
 	 * Method for authenticating token provided in request header
+	 * and checking revocation status
+	 * 
 	 * @param jwt - JWT contained in request
 	 * @return Boolean value indicating if token is authentic
 	 */
 	public static boolean isAuthentic(String jwt) {
-		// TODO implement this method
+
+		// Initialize firebase config.
+		AuthenticatorUtils.firebaseInitialize();
+
 		// authenticates provided token with Firebase
-		return true;
+		FirebaseToken decodedToken = null;
+		String uid = null;
+		try {
+			decodedToken = FirebaseAuth.getInstance().verifyIdToken(jwt);
+			System.out.println(decodedToken);
```

Comment:
either remove sysout or replace with logger.

Reply:
Sysout will be removed and re-submitted.